tidb-server/main.go

```diff
@@ -161,7 +161,6 @@ func main() {
 		sig := <-sc
 		log.Infof("Got signal [%d] to exit.", sig)
 		svr.Close()
-		os.Exit(0)
```

Comment:
If put it here, we can't make sure it be invoked at the end(this logic is in goroutine).